docs/production-deployment/cloud/get-started/api-keys.mdx

```diff
@@ -81,7 +81,7 @@ Here are the differences in their permissions:
 
 - Any user can create, delete, and update their *own* API key using the Cloud UI or `tcld`.
 - Only Global Administrators and Account Owners can create, delete, and update access to API keys for all types of Service Accounts.
-- Namespace Admins can create, delete, and update access to API keys for the Namespace Scoped Service Accounts they administer.
+- Namespace Admins can create and update access to API keys for the Namespace Scoped Service Accounts they administer.
```

Comment:
@shakeelrao just want to confirm that this still aligns with the API behavior and a non-account admin or owner **can not** delete an API key associated with a Namespace-scoped Service Account.

Reply:
https://github.com/temporalio/documentation/pull/3865

i recently refreshed the docs here and these changes reflect the current behaviour.

Reply:
Got it. I'll revert this change then and will follow-up to enable this in the UI.